.github/workflows/main.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           python -m pip --version
           python -m pip install pre-commit
           pre-commit --version
-          python -m pip install mypy==0.942
+          python -m pip install mypy==0.981
```

Comment:
Why do we need to do this here?

Reply:
Have no idea why it was there. Removed the pip install and moved the line checking mypy version after dev-requirements.